File: gc-kettle/gc-kettle-actuator/src/main/java/com/gc/kettle/actuator/properties/DatabaseMetaProperties.java
package com.gc.kettle.actuator.properties;

import lombok.Getter;
import lombok.Setter;

/**
 * @author shizhongming
 * 2020/7/16 8:33 下午
 */
@Getter
@Setter
public class DatabaseMetaProperties {

    private String type = "mysql";

    private String access = "jdbc";
    private String name = "";

    private String host = "localhost";

    private String db = "";

    private String port = "3306";
    private String dbUser = "root";
    private String dbPassword = "";

    private String resUser = "admin";

    private String resPassword = "admin";
}

File: smart-kettle/smart-kettle-common/src/main/java/com/smart/kettle/common/config/DatabaseMetaProperties.kt
package com.smart.kettle.common.config

/**
 *
 * kettle资源库连接信息
 * @author ming
 * 2019/7/3 下午8:24
 */
class DatabaseMetaProperties {

    var type = "mysql"
    var access = "jdbc"
    var name = ""
    var host: String = "localhost"

    var db: String = ""

    var port = "3306"

    var dbUser: String = "root"

    var dbPassword: String = ""

    var resUser = "admin"

    var resPassword = "admin"
}